docs/src/templates/pages/documentation/js/writing-tests.hbs

```diff
@@ -0,0 +1,304 @@
+---
+title: How to write great JavaScript unit tests
+section-title: How to write great JavaScript unit tests
+docs: true
+
+navgroup: documentation
+navsub: docs-js
+navactive: js-writing-tests
+---
+
+> ***The majority of the content here has been taken, and slightly modified, from [Marc Mignonsin's guide to unit testing in JavaScript](https://github.com/mawrkus/js-unit-testing-guide/blob/master/README.md).***
+
+<a name="contents"></a><h2 class="sg-sectionHeading">Contents</h2>
+
+1. General principles
+    1. [Unit tests](#unit-tests)
```

Comment:
@ashleynolan Just noticed that the headings below don't include the ID's that these links target 🎯

Reply:
Cool – have made those changes locally – will include them as part of my next PR as I'll be putting some more docs PRs this week 👍